lib/webpacker/file_loader.rb

```diff
@@ -7,9 +7,17 @@ class FileLoaderError < StandardError; end
   attr_accessor :data
 
   class << self
+    def exist?
+      instance.data.present? rescue false
+    end
+
     def load(path = file_path)
       self.instance = new(path)
     end
+
+    def reloadable?
+      ["development", "test"].include?(Rails.env)
```

Comment:
I think is more common use `!Rails.env.production?` in Rails code base but we can check